nuxeo-customer-project-sample-package/src/main/assemble/assembly.xml

```diff
@@ -63,86 +64,85 @@
+  &lt;update file=&quot;${package.root}/install/lib&quot; todir=&quot;${env.lib}&quot; /&gt;" />
       </then>
     </if>
 
     <var
-      name="install.properties"
-      value="${line.separator}
+            name="install.properties"
+            value="${line.separator}
   &lt;copy file=&quot;${package.root}/install/custom-package-artifacts.properties&quot;${line.separator}
     todir=&quot;${env.server.home}/sdk&quot; overwrite=&quot;true&quot; /&gt;${line.separator}
   &lt;copy file=&quot;${package.root}/install/custom-package-test-artifacts.properties&quot;${line.separator}
-    todir=&quot;${env.server.home}/sdk&quot; overwrite=&quot;true&quot; /&gt;"/>
+    todir=&quot;${env.server.home}/sdk&quot; overwrite=&quot;true&quot; /&gt;" />
 
-    <var name="install.templates" value=""/>
+    <var name="install.templates" value="" />
     <if>
-      <available file="${outdir}/marketplace/install/templates"/>
+      <available file="${outdir}/marketplace/install/templates" />
       <then>
         <var
-          name="install.templates"
-          value="${line.separator}
+                name="install.templates"
+                value="${line.separator}
   &lt;copy dir=&quot;${package.root}/install/templates&quot; todir=&quot;${env.templates}&quot; overwrite=&quot;true&quot; /&gt;${line.separator}
-  &lt;config addtemplate=&quot;custom-package&quot; /&gt;${line.separator}"/>
+  &lt;config addtemplate=&quot;custom-package&quot; /&gt;${line.separator}" />
       </then>
     </if>
 
     <copy file="src/main/resources/install.xml" todir="${outdir}/marketplace" overwrite="true">
       <filterset>
-        <filter token="BUNDLES" value="${install.bundles}"/>
-        <filter token="LIBS" value="${install.libs}"/>
-        <filter token="PROPERTIES" value="${install.properties}"/>
-        <filter token="TEMPLATES" value="${install.templates}"/>
+        <filter token="BUNDLES" value="${install.bundles}" />
+        <filter token="LIBS" value="${install.libs}" />
+        <filter token="PROPERTIES" value="${install.properties}" />
+        <filter token="TEMPLATES" value="${install.templates}" />
       </filterset>
     </copy>
 
     <!-- Studio Registries Extraction -->
     <nx:studioExtraction todir="${outdir}/marketplace">
-      <fileset dir="${outdir}/marketplace/install/bundles"/>
+      <fileset dir="${outdir}/marketplace/install/bundles" />
     </nx:studioExtraction>
 
-    <zip destfile="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" basedir="${outdir}/marketplace"/>
-    <artifact:attach file="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" type="zip"/>
+    <zip destfile="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip"
+         basedir="${outdir}/marketplace" />
+    <artifact:attach file="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" type="zip" />
   </target>
```

Comment:
could add the cleanup: https://github.com/nuxeo/nuxeo-explorer/blob/master/packages/src/main/assemble/assembly.xml#L129-L133